File: src/games/CemuGame.js
const { WiiUEmulationGame } = require("./WiiUEmulationGame.js");

/**
 * A class representing a cemu (in lutris) game
 */
class CemuGame extends WiiUEmulationGame{
	constructor(name, path){
		super(name, path);
	}
}

module.exports = {
	CemuGame
};

File: gali/sources/cemu/cemu_game.py
from dataclasses import dataclass, field

from gali.sources.emulation_game import EmulationGame
from gali.sources.cemu.cemu_lutris_startup_chain import CemuLutrisStartupChain


@dataclass
class CemuGame(EmulationGame):
    """Abstract class representing a Cemu game"""

    platform: str = field(default="Nintendo - Wii U", init=False)


@dataclass
class CemuLutrisGame(CemuGame):
    """Class representing a Cemu in Lutris game"""

    wine_prefix_path: str = field(default="")
    cemu_slug: str = field(default="cemu")
    startup_chains = [
        CemuLutrisStartupChain
    ]
